README.md

```diff
@@ -6,16 +6,17 @@ You can find NuGet packages for these libraries [here](packages.md).
 
```

Comment:
Shouldn't it be Azure instead of Microsoft Azure?

Reply:
will change